PowerShell.sln

```diff
@@ -123,14 +125,6 @@ Global
 		{439A24FC-8E0A-48B6-8227-44C297311F49}.Release|Any CPU.Build.0 = Release|Any CPU
 		{07BFD271-8992-4F34-9091-6CFC3E224A24}.CodeCoverage|Any CPU.ActiveCfg = CodeCoverage|Any CPU
 		{07BFD271-8992-4F34-9091-6CFC3E224A24}.CodeCoverage|Any CPU.Build.0 = CodeCoverage|Any CPU
-		{07BFD271-8992-4F34-9091-6CFC3E224A24}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{07BFD271-8992-4F34-9091-6CFC3E224A24}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{07BFD271-8992-4F34-9091-6CFC3E224A24}.Linux|Any CPU.ActiveCfg = Linux|Any CPU
-		{07BFD271-8992-4F34-9091-6CFC3E224A24}.Linux|Any CPU.Build.0 = Linux|Any CPU
-		{07BFD271-8992-4F34-9091-6CFC3E224A24}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{07BFD271-8992-4F34-9091-6CFC3E224A24}.Release|Any CPU.Build.0 = Release|Any CPU
-		{8F63D134-E413-4181-936D-D82F3F5F1D85}.CodeCoverage|Any CPU.ActiveCfg = CodeCoverage|Any CPU
-		{8F63D134-E413-4181-936D-D82F3F5F1D85}.CodeCoverage|Any CPU.Build.0 = CodeCoverage|Any CPU
```

Comment:
You removed these two entries that are associated with `8F63D134-E413-4181-936D-D82F3F5F1D85`, but left two entries associated with `07BFD271-8992-4F34-9091-6CFC3E224A24` above. Is that intentional?

Reply:
I believe it is better to rebase then try to resolve conflicts in such auto changed file.

Reply:
The conflict is already resolved and I resolved it by taking HEAD and adding the project again to the solution

Reply:
@bergmeister Please take a look at the highlighted entries in the screenshot below. It looks to me you shouldn't remove the two `8F63D134` entries.

![image](https://user-images.githubusercontent.com/127450/42834892-b45ef702-89ac-11e8-8efb-2b029429dc0b.png)

Reply:
@daxian-dbw Sorry, thanks for reminding me to address your comment as well. Yes, I made 3 commits to address this:
- Remove the 2 redundant `07BFD271` lines
- Revert the removal of the 2 `8F63D134` lines
- Reorder lines so that VS does not automatically re-order them on save,